src/commands/cmd_bloom_filter.cc

```diff
@@ -104,14 +105,86 @@ class CommandBFExists : public Commander {
     redis::BloomChain bloom_db(svr->storage, conn->GetNamespace());
     int ret = 0;
     auto s = bloom_db.Exist(args_[1], args_[2], &ret);
+    if (s.IsNotFound()) return {Status::RedisExecErr, "key is not found"};
     if (!s.ok()) return {Status::RedisExecErr, s.ToString()};
 
     *output = redis::Integer(ret);
     return Status::OK();
   }
 };
 
+class CommandBFInfo : public Commander {
+ public:
+  Status Parse(const std::vector<std::string> &args) override {
+    CommandParser parser(args, 2);
```

Comment:
We'd better `args` should not greater than 3?

https://github.com/RedisBloom/RedisBloom/blob/master/src/rebloom.c#L949